.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "blt"]
 	path = blt
 	url = https://github.com/LLNL/blt.git
+	ignore = dirty
```

Comment:
Why is this change here?  Is this project-wide change necessary to support caliper?  It does not seem to be related?

Reply:
@mcfadden8 Thanks for catching .gitmodules issue. I was probably advancing BLT manually early on in the project. I'll clean it up